internal/services/network/express_route_connection_resource_test.go

```diff
@@ -223,6 +223,7 @@ resource "azurerm_express_route_connection" "test" {
   routing_weight                       = 2
   authorization_key                    = "90f8db47-e25b-4b65-a68b-7743ced2a16b"
   enable_internet_security             = true
+  enable_private_link_fast_path        = true
```

Comment:
This should also be added to the `complete` test config.

Reply:
handled in [fc10726](https://github.com/hashicorp/terraform-provider-azurerm/pull/25596/commits/fc107262eb9afccf873dd7af1395458311eb17e2)